tools/PI/DevHome.PI/BarWindow.xaml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="utf-8"?>
+<winex:WindowEx
+    x:Class="DevHome.PI.BarWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:winex="using:WinUIEx"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:controls="using:DevHome.PI.Controls"
+    xmlns:helpers="using:DevHome.PI.Helpers"
+    mc:Ignorable="d"
+    Title="" MinHeight="0" MinWidth="0" Height="0" Width="0"
+    TaskBarIcon="Images/pi.ico" IsTitleBarVisible="False"
+    Closed="WindowEx_Closed">
+
+    <!-- TODO Make DesktopAcrylicBackdrop/MicaBackdrop and AcrylicBackgroundFillColorBaseBrush user-configurable.-->

```

Comment:
nit: can probably just remove this. Mica is usually just controlled system wide and the SystemBackdrop takes care of that for us.